packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -230,41 +245,32 @@
 +      - effect: "NoSchedule"
 +        operator: "Exists"
 +
-+## Component scraping nginx-ingress-controller
-+##
-+ingressNginx:
++rke2IngressNginx:
 +  enabled: false
-+
-+  ## The namespace to search for your nginx-ingress-controller
-+  ##
-+  namespace: ingress-nginx
-+  
-+  service:
-+    port: 9913
-+    targetPort: 10254
```

Comment:
We cannot make a breaking change to an already released chart, so `ingressNginx` should still exist / be respected.

I think we should follow the pattern of https://github.com/rancher/charts/pull/1245, where:
1. We cannot set `rkeIngressNginx.enabled=true` unless `ingressNginx.enabled=true`. Fail on create if so.
2. If `rkeIngressNginx.enabled=true`, we don't deploy the ServiceMonitor or Service for `ingressNginx`
3. No need to change the ConfigMap since `rkeIngressNginx.enabled` implies `ingressNginx.enabled`.

Reply:
changes are made based on your comments.